automated-tests/src/EMBC.Tests.Automation/EMBC.Tests.Automation.Responders/StepDefinitions/RespondersPortalSteps.cs

```diff
@@ -3,268 +3,279 @@
     [Binding]
     public sealed class RespondersPortalSteps
     {
-        private readonly EvacueeRegistration evacueeRegistration;
+        private readonly AssignTask assignTask;
+        private readonly SearchEvacuee searchEvacuee;
+        private readonly ESSFileDashboard essFileDashboard;
+        private readonly WizardProfile wizardProfile;
+        private readonly WizardEssFile wizardEssFile;
+        private readonly WizardSupport wizardSupport;
 
         public RespondersPortalSteps(BrowserDriver driver)
         {
-            this.evacueeRegistration = new EvacueeRegistration(driver.Current);
+            
+            this.assignTask = new AssignTask(driver.Current);
```

Comment:
You may want to consider splitting steps into multiple classes as well, scenarios can all share them when they compose the steps